src/Persistence/EntityFramework/Migrations/00000000000000_Initial.cs

```diff
@@ -70,6 +70,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 {
                     Id = table.Column<Guid>(nullable: false),
                     MaximumLevel = table.Column<short>(nullable: false),
+                    PKClearZenCost = table.Column<int>(nullable: false, defaultValueSql: "300000"),
```

Comment:
Please remove the defaultValueSql. Instead, add the assignment of the initial value in the DataInitialization.InitializeGameConfiguration() function.

There are also more places where it needs to be added:
* EntityDataContextModelSnapshot
* 00000000000000_Initial.designer.cs